docs/source/algorithms.rst

```diff
@@ -184,6 +184,73 @@ processor:
 
 When a `primitive` is not available on a processor, a conversion occurs automatically after the computation is complete.
 
+Batch jobs
+++++++++++
+
+The ``Sampler`` can setup a batch of different sampling tasks within a single job. Such a job enables you to gain
+some time (overhead of job management) as well as tidy up your job list, especially when running on the Quandela Cloud
+(but it can still be used in a local simulation context).
+
+The system relies on defining a circuit containing variable parameters, then with each iteration of the batch job,
+different values of these parameters can be set, along the input state and the `detected photons filter`. Each iteration
```

Comment:
Pourquoi detected photons filter est en italique et pas le reste ? (comme input state par exemple ?)

Reply:
Changed to a bullet list + all terms are in italics